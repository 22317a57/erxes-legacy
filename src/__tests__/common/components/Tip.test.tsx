import { mount, shallow } from 'enzyme';
import React from 'react';

import Button from '../../../modules/common/components/Button';
import Tip from '../../../modules/common/components/Tip';

describe('FilterToggler component', () => {
  const defaultProps = {
    children: <Button />
  };

  test('renders FilterToggler successfully', () => {
    shallow(<Tip {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<Tip {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });
});
