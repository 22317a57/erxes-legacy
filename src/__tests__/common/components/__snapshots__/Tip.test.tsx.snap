// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterToggler component snapshot matches 1`] = `
<button
  className="sc-bZQynM jGbjye"
  onBlur={[Function]}
  onClick={null}
  onFocus={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  type="button"
/>
`;
